arroyo/backends/kafka/consumer.py

```diff
@@ -269,6 +270,10 @@ def subscribe(
         def assignment_callback(
             consumer: ConfluentConsumer, partitions: Sequence[ConfluentTopicPartition]
         ) -> None:
+            if not partitions and self.__assignment_strategy == "cooperative-sticky":
+                logger.info("skipping empty assignment")
+                return
```

Comment:
Why do you need a different logic for partition assignment between cooperative and standard in case of empty assignment? 
I assume you can get an empty assignment in the cooperative rebalancing when, after a rebalancing, your assignments do not change. Is that the scenario where you do not want to touch the existing assignments ?

Reply:
after sleeping on it i agree. i only added this because it made cooperative rebalancing more comprehensible, and wasn't sure of the implications on regular rebalancing. i think we can skip empty assignments regardless of the assignment strategy.